### PR TITLE
Update zh-rCN strings.xml

### PR DESCRIPTION
**Pull request by cccClyde**
Made some refinements for Simplified Chinese localization based on native users' feedback.

### CODE REVIEW

**Comment by danilkinkin** [comment]
Please fix me if I incorrect understand. Now in file used to simplified Chinese and you want to change to traditional Chinese?

Android offers these locale options. If these are different languages, it is better to put them in different files 


Locale Name | locale code | variant (country) | values folder name
-- | -- | -- | --
Chinese | zh |   | values-zh
Chinese (Simplified Han) | zh |   | values-zh-r_#Hans
Chinese (Simplified Han,China) | zh | CN | values-zh-rCN_#Hans
Chinese (Simplified Han,Hong Kong) | zh | HK | values-zh-rHK_#Hans
Chinese (Simplified Han,Macau) | zh | MO | values-zh-rMO_#Hans
Chinese (Simplified Han,Singapore) | zh | SG | values-zh-rSG_#Hans
Chinese (Traditional Han) | zh |   | values-zh-r_#Hant
Chinese (Traditional Han,Hong Kong) | zh | HK | values-zh-rHK_#Hant
Chinese (Traditional Han,Macau) | zh | MO | values-zh-rMO_#Hant
Chinese (Traditional Han,Taiwan) | zh | TW | values-zh-rTW_#Hant

**Comment by cccClyde** [comment]
> Please fix me if I incorrect understand. Now in file used to simplified Chinese and you want to change to traditional Chinese?
> 
> Android offers these locale options. If these are different languages, it is better to put them in different files
> 
> Locale Name	locale code	variant (country)	values folder name
> Chinese	zh	 	values-zh
> Chinese (Simplified Han)	zh	 	values-zh-r_#Hans
> Chinese (Simplified Han,China)	zh	CN	values-zh-rCN_#Hans
> Chinese (Simplified Han,Hong Kong)	zh	HK	values-zh-rHK_#Hans
> Chinese (Simplified Han,Macau)	zh	MO	values-zh-rMO_#Hans
> Chinese (Simplified Han,Singapore)	zh	SG	values-zh-rSG_#Hans
> Chinese (Traditional Han)	zh	 	values-zh-r_#Hant
> Chinese (Traditional Han,Hong Kong)	zh	HK	values-zh-rHK_#Hant
> Chinese (Traditional Han,Macau)	zh	MO	values-zh-rMO_#Hant
> Chinese (Traditional Han,Taiwan)	zh	TW	values-zh-rTW_#Hant

No. This PR is intended for Simplified Chinese, specificly `values-zh`, `values-zh-r_#Hans`, `values-zh-rCN_#Hans`, `values-zh-rHK_#Hans`, `values-zh-rMO_#Hans` and `values-zh-rSG_#Hans`. 

Personally I choose `values-zh-rCN_#Hans` but in fact the same file can be applied to other variants of Simplified Chinese. The difference between these locales is almost nothing, just for Simplified Chinese users from different countries or regions to choose from.

In short, this PR is for Simplified Chinese. You can actually put it to all Simplified Chinese folders but NOT traditional ones.

**Comment by cccClyde** [comment]
> Ok, maybe I understood. You made translation for simplified Chinese and what dialect of Chinese usage now? Traditional? I need override or put current translation to file with traditional, and your in simplified?

No it's not a dialect or somethings related to traditional. **Just overwrite your current Simplified Chinese strings** (located in `buckwheat/app/src/main/res/values-zh-rCN`) with this one and you're good to go.

As for those other variants of Simplified Chinese, you can create some new folders (values-zh, values-zh-r, values-zh-rHK, values-zh-rMO and values-zh-rSG) and put the same file in them. They're all Simplified Chinese and can use the same strings file.

**Comment by danilkinkin** [comment]
Ok, I understood and will change. And lastly, can you say why difference so big between some strings? Looks like different language (I ask only for interest)

For example:
`<string name="app_widget_extend_name">Buckwheat 大</string>`
`<string name="app_widget_extend_name">Buckwheat 扩展</string>`

**Comment by cccClyde** [comment]
> Ok, I understood and will change. And lastly, can you say why difference so big between some strings? Looks like different language (I ask only for interest)
> 
> For example: `<string name="app_widget_extend_name">Buckwheat 大</string>` `<string name="app_widget_extend_name">Buckwheat 扩展</string>`

Gladly. Like I wrote in the first reply, everything is just for refinement. Localization is far more than just machine translating strings from one language to anther, otherwise it would cause confusion and misunderstanding.

Take the example you brought up for instance, in the case where user is reading explanations of widgets, it is okay to just tell them this one is "a big widget of Buckwheat". But if the same string translated without any consideration of the context, the meaning of the string becomes ambiguous, since the word "big" can be used to describe multiple aspects even when talking about widget, font, background, padding... So I chose a more common phrase in Chinese which also representing "big", but also indicating "an extended layout" for Simplified Chinese users to easily understand.

In short, translating "Buckwheat big" into "Buckwheat 大" was an easy take and it's not wrong, but the final result just looks fracture.

Some other examples including where I translated "spending" with "支出" instead of ''花费''. Because ''支出" is a more formal way to indicate spending in Chinese. "花费", however is more like "cost" in English - time, money, energy... It's not wrong here again, but it's jut not the best choice.

**Comment by danilkinkin** [comment]
Wow. Thanks, it's so interesting. Big thanks for your work
